web-app/web-frontend/script.js

```diff
@@ -72,12 +37,46 @@ function onSnapEnd () {
   board.position(game.fen())
 }
 
+function updateStatus () {
+  var status = ''
+
+  var moveColor = 'White'
+  if (game.turn() === 'b') {
+    moveColor = 'Black'
+  }
```

Comment:
It might be nice to use a [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) here so it's clear what's going on. `var moveColor = (game.turn() == 'b') ? "Black" : "White";`

Reply:
Thank you! Ok I will work on that!

Reply:
Thanks looks great!